api/features/tests/test_tasks.py

```diff
@@ -30,16 +27,22 @@ def test_trigger_feature_state_change_webhooks(MockThread):
     feature_state.feature_state_value.save()
     feature_state.save()
 
-    MockThread.reset_mock()  # reset mock as it will have been called when setting up the data
+    mock_call_environment_webhooks = mocker.patch(
+        "features.tasks.call_environment_webhooks.delay"
+    )
+    mock_call_organisation_webhooks = mocker.patch(
+        "features.tasks.call_organisation_webhooks.delay"
+    )
 
     # When
     trigger_feature_state_change_webhooks(feature_state)
 
     # Then
-    call_list = MockThread.call_args_list
+    environment_webhook_call_args = mock_call_environment_webhooks.call_args
+    organisation_webhook_call_args = mock_call_organisation_webhooks.call_args
 
-    environment_webhook_call_args = call_list[0]
-    organisation_webhook_call_args = call_list[1]
+    assert environment_webhook_call_args[1]["args"][0] == environment.id
+    assert organisation_webhook_call_args[1]["args"][0] == organisation.id
```

Comment:
This confused me for a while until I ran the test myself. I think it's because the kwarg we pass to the `delay` method is called `args` (maybe that was a bad decision). I wonder if we can improve this with some better naming of vars (and using the attributes on the `Call` class via their names too) here though, something like: 

```python
environment_webhook_call_args = (
    mock_call_environment_webhooks.call_args.kwargs["args"]
)
organisation_webhook_call_args = (
    mock_call_organisation_webhooks.call_args.kwargs["args"]
)
```

If you agree, we should also do this in similar tests below.

Reply:
Agreed, did the changes and it is looking much more readable now. Thanks!